elytron/src/main/java/org/wildfly/extension/elytron/KeyStoreService.java

```diff
@@ -176,6 +176,14 @@ public void start(StartContext startContext) throws StartException {
                         keyStore = AtomicLoadKeyStore.atomize(detected);
                     }
                 } else {
+                    if (keyStore == null) {
+                        ROOT_LOGGER.debugf(
```

Comment:
Don't change unless you feel you want to as this PR does look good as it is.

You could immediately before the debugf call add a line like:

String type = KeyStore.getDefaultType();

This would mean you could include the type in the debug message as well.  You could then use this String at the point the KeyStore is created to avoid a second call to getDefaultType.

But this is only a minor consideration, I am happy to approve as is.  Thank you for the changes.

Reply:
Hi @darranl, commit added :-) Thanks